CODEOWNERS

```diff
@@ -13,6 +13,7 @@
 /specification/billing/ @wilcobmsft @asarkar84
 /specification/cdn/ @jorinmejia @yunhemsft @jessicl-ms @sinadell @rrahulms @t-bzhan
 /specification/cognitiveservices/ @felixwa @yangyuan
+/specification/cognitiveservices/data-plane/Language/ @rokulka @ChongTang @annatisch @heaths
```

Comment:
Would you add team `@Azure/api-stewardship-board` to owner as well, so that it automatically loop-in reviewers from API Stewardship for possible async review.

Reply:
added.